src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -328,13 +329,25 @@ public function url($path)
      *
      * @param  \League\Flysystem\AwsS3v3\AwsS3Adapter  $adapter
      * @param  string  $path
+     * @param  string  $expires
      * @return string
      */
-    protected function getAwsUrl($adapter, $path)
+    protected function getAwsUrl($adapter, $path, $expires)
```

Comment:
I don't think this parameter should be optional as the public one has already been made optional.